components/compass-runtime-agent/internal/compassconnection/supervisor.go

```diff
@@ -172,14 +172,14 @@ func (s *crSupervisor) SynchronizeWithCompass(connection *v1alpha1.CompassConnec
 		s.log.Info(res)
 	}
 
-	s.log.Infof("Labeling Runtime with URLs...")
-	_, err = directorClient.SetURLsLabels(s.runtimeURLsConfig)
```

Comment:
There are still some searchable references of SetURLsLabels in the code, especially in mocks that require regeneration.

Reply:
I forgot to regenerate the mocks, I'm ashamed of myself... 😆 I'll correct this immediately.